Stage1/environment/env_1d.py

```diff
@@ -2,7 +2,37 @@
 import cv2
 import os
 import gym
+import pyglet
 from gym import spaces
+from gym.envs.classic_control import rendering
+
+
+class ImageObject:
+    def __init__(self, bites):
+        self.bites = bites
+
+    def read(self):
+        return self.bites
+
+
+class NPImage(rendering.Geom):
+    def __init__(self, fname, array_img, width, height):
+        rendering.Geom.__init__(self)
+        self.extension = os.path.splitext(fname)[1].lower()
+        self.set_color(1.0, 1.0, 1.0)
+        self.width = width
+        self.height = height
+        file = self.conv_array_to_bytes(array_img)
+        img = pyglet.image.load(fname, file=file)
+        self.img = img
+        self.flip = False
+
+    def render1(self):
+        self.img.blit(0, 0, width=self.width, height=self.height)
+
+    def conv_array_to_bytes(self, array: np.array):
```

Comment:
Please use clear names. What does "conv" mean? If it is "convert", then name it "convert_...".